SparkPost/2web2ui
# SA-2019: Fixed issue with delay report tab not properly using filters

### What Changed

- Delay tab in report builder not properly attaching request filters

### How To Test

- Go to report builder
- Add metrics relating to delays
- Go to delay tab in report builder
- Verify request has no filters attached
- Add filters
- Verify request has filters attached

### To Do
